core/codegen/src/lib.rs

```diff
@@ -1,6 +1,5 @@
 #![feature(proc_macro_diagnostic, proc_macro_span)]
 #![feature(crate_visibility_modifier)]
-#![feature(transpose_result)]
```

Comment:
This bumps the minimum rustc for `codegen`, but only to around 2019-01-13. Since there are a few other PRs open that affect the minimum rustc, we can just take care of this when merging to avoid conflicts.